src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +23,22 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set<Character> uniqueCharsSet = new TreeSet<>();
+        StringBuilder result = new StringBuilder();
+        BufferedReader reader = new BufferedReader(new FileReader(fileName));
+
+        int symbol = reader.read();
+        while (symbol > 0) {
+            if (Character.isAlphabetic(symbol)) {
```

Comment:
```suggestion
            if (Character.isLetter(symbol)) {
```
check out the difference)